src/moveit_base.cpp

```diff
@@ -236,21 +236,24 @@ moveit::core::RobotStatePtr MoveItBase::getCurrentState()
   return current_state_;
 }
 
-bool MoveItBase::getTFTransform(const std::string& from_frame, const std::string& to_frame, Eigen::Affine3d &pose)
+bool MoveItBase::getTFTransform(const std::string& from_frame, const std::string& to_frame, Eigen::Isometry3d &pose)
```

Comment:
I actually don't think this function belongs in the boilerplate, let's just remove it.